Backtracking/generateParentheses.js

```diff
@@ -0,0 +1,31 @@
+/**
+ *
+ * Problem Statement: Given a number n pairs of parentheses, try to Generate all combinations of valid parentheses;
+ *
+ * Link to problem: https://leetcode.com/problems/generate-parentheses/
+ */
+
+const generateParentheses = (n) => {
+  const res = []
+
+  const solve = (n, chres, openParenthese, closedParenthese, res) => {
```

Comment:
You don't need to pass `res` and `n` as params since `solve` is a closure

Reply:
Done !